src/Main.java

```diff
@@ -1,5 +1,48 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+//Task 1
+        System.out.println("Задача №1");
+        int[] arr = {10, 25, 16, 9, 20};
+        int sum = 0;
+        for (int i = 0; i < arr.length; i++) {
+            sum += arr[i];
+        }
+        System.out.println("«Сумма трат за месяц составила " + sum + " рублей»");
+
+
+//Task 2
+        System.out.println("Задача №2");
+        int[] arr2 = {25, 15, 10, 30, 20};
+        int arr2Max =Integer.MIN_VALUE ;
```

Comment:
Еще вариант приравнять к первому значению массив
```suggestion
        int arr2Max = arr[0];
```